src/the_way/filter.rs

```diff
@@ -60,6 +65,19 @@ impl TheWay {
             }
             (None, Some(snippets)) => Snippet::filter_in_date_range(snippets, from_date, to_date),
             (None, None) => self.list_snippets_in_date_range(from_date, to_date),
-        }
+        };
+        let snippets = match &filters.pattern {
+            Some(pattern) => {
+                let regex = Regex::new(&pattern.to_string_lossy())?;
+                snippets.map(|snippets| {
+                    snippets
+                        .into_iter()
+                        .filter(|snippet| regex.is_match(&snippet.description))
```

Comment:
This is a neat feature! Would be nice if the regex also considers the tags and code instead of just the description (like the interactive search currently does) - could you add them in as extra conditions here?

Reply:
Sure, added.

Reply:
Great, I'll merge and make a new release, thanks.